src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -1823,6 +1823,23 @@ public void run() {
                 .itemOutputs(ItemList.Machine_Multi_LargeChemicalReactor.get(1)).duration(10 * SECONDS)
                 .eut(TierEU.RECIPE_HV).addTo(assemblerRecipes);
 
+        // Industrial Autoclave
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        ItemList.AutoclaveLuV.get(2L),
+                        ItemList.Casing_Autoclave.get(4L),
+                        GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.LuV, 4L),
+                        GT_ModHandler.getModItem(GTPlusPlus.ID, "itemPlateLafiumCompound", 8L))
+                .fluidInputs(Materials.Polybenzimidazole.getMolten(8 * 144))
+                .itemOutputs(ItemList.Machine_Multi_Autoclave.get(1L)).duration(30 * SECONDS).eut(TierEU.RECIPE_LuV)
+                .addTo(assemblerRecipes);
+
+        // Industrial Autoclave
+        GT_Values.RA.stdBuilder()
```

Comment:
this recipe looks like it produce nothing. Where is the output item/blocks ? @Volence

Reply:
@Dream-Master Line 1840 has `.itemOutputs(ItemList.Casing_Autoclave.get(1L))`

Reply:
ah ok